templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -1182,16 +1164,13 @@ table {
     margin-left: auto;
   }
 }
-
 .indented-options {
   border-left: solid 2px #ccc;
   padding-left: 1em;
 }
-
 button:disabled {
   cursor: not-allowed;
 }
-
```

Comment:
Why are we removing all these empty lines suddenly from this file?

Reply:
My bad...I forget how that happened. I reverted the changes.